health-integrations/README.md

```diff
@@ -14,3 +15,27 @@ InjectorBuilder
     )
     .createInjector();
 ```
+
+### Archaius2-backed IndicatorFilter
+Filtering of which HealthIndicators will be used in communicating the health status of the application is provided via an [Archaius2](https://github.com/Netflix/archaius/tree/2.x) implementation of [IndicatorFilter](https://github.com/Netflix/runtime-health/blob/addIndicatorFilterSupport/health-api/src/main/java/com/netflix/runtime/health/api/IndicatorFilter.java).
+
+```
+InjectorBuilder
+    .fromModules(
+        // ... other modules ...
+        new HealthModule(),
+        new EurekaHealthStatusBridgeModule(),
+        new ArchaiusHealthStatusFilterModule(),
+        // ... other modules ...
+    )
+    .createInjector();
+```
+
+Indicators can now be included/excluded by setting the following properties in Archaius2.
+```
+health.status.indicators.include=com.myproject.MyFilter
+health.status.indicators.exclude=com.myproject.DontIncludeThisFilter
```

Comment:
Isn't there no situation in which you'd specify both include and exclude?  It seems to me if I have just the first line here `health.status.indicators.include=com.myproject.MyFilter` that everything would be excluded that wasn't com.myproject.MyFilter?